src/Forms/TextField.php

```diff
@@ -16,6 +16,11 @@ class TextField extends FormField
 
     protected $schemaDataType = FormField::SCHEMA_DATA_TYPE_TEXT;
 
+    /**
+     * @var Tip|null A tip to render beside the input
+     */
+    protected $tip;
```

Comment:
I think it would be fair to make this private and expect any sub-class to use or override the getter/setter if they need to do weird things.